lib/config.js

```diff
@@ -24,7 +31,9 @@ module.exports.getConfig = async function getConfig({ context, configName }) {
       log({
         context,
         message:
-          'Config validation errors, please fix the following issues in release-drafter.yml:\n' +
+          'Config validation errors, please fix the following issues in ' +
```

Comment:
If the error relates to the file on the default branch, then this error message should mention that.

Reply:
That is done exactly here:
https://github.com/release-drafter/release-drafter/blob/0bc3bc59906b1dac1b608cc7ff5441fcb86ad6c9/lib/config.js#L15-L21

Reply:
That's if it's missing.

If it's found and invalid, which branch did it read?

Reply:
Then joiValidationErrorsAsTable(error) will print out the errors?

Reply:
But will it mention the branch?

Reply:
We can't really know which "branch" as the config gets merged. So it could be in default branch or on the `.github` repo you need to fix the config.

Reply:
Ah.

Could I convince you to reparse the configurations separately when you get an error? This wouldn't slow down the default path (beyond a slightly larger file), but would result in a much better user experience.

Reply:
I don't want to create a custom version of https://github.com/probot/octokit-plugin-config to satisfy slightly better user experience. User should be able to comprehend release-drafter config setup. They are the once setting it up. You can keep it as simple as you want or as complex as you want.

Again potentially you could have endless `_extends` like 10 levels deep if you wanted.